images/router/haproxy/conf/haproxy-config.template

```diff
@@ -131,7 +131,8 @@ defaults
   # backend service or alternatively, you can send a custom 503 error.
```

Comment:
Please update the comment here regarding custom error pages.  Is it still correct that one can uncomment the line?